app/views/api/v0/classified_listings/index.json.jbuilder

```diff
@@ -0,0 +1,36 @@
+json.array! @classified_listings do |listing|
+  json.type_of              "listings"
+  json.id                   listing.id
+  json.title                listing.title
+  json.slug                 listing.slug
+  json.body_markdown        listing.body_markdown
+  json.bumped_at            listing.bumped_at
```

Comment:
not sure we want this info to be public, @benhalpern ?

Reply:
I am sorry, I am a bit confused as I cannot figure which field it is. Is it json.id here ?

Reply:
`bumped_at` should not be in the JSON I think